#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_dir_seperator.dpatch by  <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix dir seperator for POSIX

@DPATCH@

diff -ur nunit-2.2.6/src/ClientUtilities/util/nunit.util.build nunit-2.2.6-new/src/ClientUtilities/util/nunit.util.build
--- nunit-2.2.6/src/ClientUtilities/util/nunit.util.build	2005-12-09 08:08:12.000000000 +1300
+++ nunit-2.2.6-new/src/ClientUtilities/util/nunit.util.build	2006-01-24 12:39:00.000000000 +1300
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <project name="NUnit.Util" default="build">
 
-  <include buildfile="..\..\nunit.project.include"/>
+  <include buildfile="../../nunit.project.include"/>
   
   <property name="current.build.output" value="nunit.util"/>
 
